docs/docs/forms.mdx

```diff
@@ -360,74 +360,101 @@ to fully customize the validation process.
 
 If none of the predefined [Slot Mappings](./forms.mdx#slot-mappings) fit your use
 case, you can use the
-[Custom Action](./actions.mdx#custom-actions) `validate_{form_name}` to write your own
-extraction code. Rasa Open Source will trigger this function when the form is run.
+[Custom Action](./actions.mdx#custom-actions) `validate_<form_name>` to write your own
+extraction code. Rasa Open Source will trigger this action when the form is run.
 
-Make sure your custom action returns `SlotSet` events for every extracted value.
-The following example shows the implementation of a custom slot mapping which sets
-a slot based on the length of the last user message.
+If you're using the Rasa SDK we recommend you to extend the provided
+`FormValidationAction`. When using the `FormValidationAction`, two things are needed to
+extract customs slots:
+
+1. Define a method `extract_<slot_name>` for every custom slot mapping which you want to
+   extract.
+2. Optional: If a slot is not part of your mapped slots in the domain,
+   override `required_slots` to make sure the additional slot is considered by the form.
```

Comment:
Wait, currently we want users to **not** include in the domain (for the given form) any slots that should use custom mappings, right? So this 2nd point should be more like two different points:
```suggestion
2. Make sure that in the domain file you list for your form only those slots that don't use custom mappings.
3. Override `required_slots` to add all slots with custom mappings to the list of slots the form should request.
```